### PR TITLE
Feature/quiz

### PR DESCRIPTION
**Pull request by DavidWilliamK**

### CODE REVIEW

File: src/main/java/com/future/function/qa/pages/scoring/Quizzes/QuizFormPage.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    public WebElementFacade calendarWeek(int row) {
+        return find(By.xpath(String.format("//body/div[@class='app']/div[@class='main-container is-fullhd']" +
+                "/div[@class='main-content']/div[@class='auto-overflow-container']/div[@class='quiz-form__container']" +
+                "/div[@class='quiz-form__container-details']/div[@class='quiz-form__container-details__date']" +
+                "/div[@class='quiz-form__container-details-input']/div[@class='field is-floating-label']" +
+                "/div[@class='datepicker control']/div[@class='dropdown is-active is-mobile-modal']" +
+                "/div[@class='dropdown-menu']/div[@class='dropdown-content']/div[@class='dropdown-item']" +
+                "/div[@class='datepicker-content']/section[@class='datepicker-table']" +
+                "/div[@class='datepicker-body']/div[%s]", row + 1)));
+    }
```
Comment (oliverrsebastian) [inline src/main/java/com/future/function/qa/pages/scoring/Quizzes/QuizFormPage.java:34]: Panjang bener

File: src/test/java/com/future/function/qa/steps/Scoring/Landing/GradesSteps.java [hunk 2]
```diff
@@ -75,6 +75,11 @@ public void userShouldNotSeeAssignmentActionDropdownButtonOnTabItemWithIndex(int
         gradesPage.findGradesItemActionDropdownButton(tabIdx, row).shouldNotBeVisible();
     }
 
+    @When("^user clicks action dropdown button on tab (\\d+) item with index (\\d+)$")
+    public void userClicksAssignmentActionDropdownButtonOnTabItemWithIndex(int tabIdx, int row) {
```
Comment (oliverrsebastian) [inline src/test/java/com/future/function/qa/steps/Scoring/Landing/GradesSteps.java]: Remove Assignment from method name

File: src/test/java/com/future/function/qa/steps/Scoring/Quiz/QuizDetailSteps.java [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+    @And("^user should see quiz description$")
+    public void userShouldSeeQuizDescription() {
+        detailPage.quizDescription().shouldBeVisible();
+    }
+
+    @And("^user should see quiz' \"([^\"]*)\" on detail box number (\\d+)$")
+    public void userShouldSeeQuizOnDetailBoxNumber(String boxTitle, int col) {
+        detailPage.quizDetailBox(col).shouldBeVisible();
+        assertTrue(detailPage.detailBoxHeader(col).getText().contains(boxTitle));
```
Comment (oliverrsebastian) [inline src/test/java/com/future/function/qa/steps/Scoring/Quiz/QuizDetailSteps.java:81]: `contains` can be change to `equals` if you want to exactly compare the strings

File: src/main/java/com/future/function/qa/pages/scoring/Quizzes/QuizFormPage.java (hunk 1 above)
Comment (DavidWilliamK) [reply src/main/java/com/future/function/qa/pages/scoring/Quizzes/QuizFormPage.java:34]: Ini testing buat component bawaan dari buefy per @oliverrsebastian

File: src/test/java/com/future/function/qa/steps/Scoring/Quiz/QuizDetailSteps.java (hunk 3 above)
Comment (DavidWilliamK) [reply src/test/java/com/future/function/qa/steps/Scoring/Quiz/QuizDetailSteps.java:81]: This won't work
The reason being `detailBoxHeader` contains 2 texts; title and value

**Review by oliverrsebastian — APPROVED** [review]

**Pull request merged**